packages/winpanda/extra/src/winpanda/common/utils.py

```diff
@@ -95,3 +97,36 @@ def rmdir(path, recursive=False):
             LOG.debug(f'rmdir(): Remove directory: {path_}')
     else:
         LOG.debug(f'rmdir(): Path not found: {path_}')
+
+
+def run_external_command(cl_elements, timeout=30):
+    """Run external command.
+
+    :param cl_elements: list, elements of command line, beginning with
+                        executable name
+    :param timeout:     int|float, forcibly terminate execution, if this number
+                        seconds passed since execution was started
+    :return:            subprocess.CompletedProcess, results of sub-process
+                        execution
+    """
+    # cl_elements = cl_elements if isinstance(cl_elements, list) else []
+    timeout = timeout if isinstance(timeout, (int, float)) else 30
```

Comment:
The only way this takes the `else` path is if the caller passes a non-numeric parameter. This should be allowed to fail rather than hiding the problem.